HW_16/HW_16.py

```diff
@@ -0,0 +1,85 @@
+website = "https://djinni.co/"
+
+# Ancestor
+xpath_selector_1 = "//a[@data-analytics-param='login_homepage']/ancestor::li"

```

Comment:
![image](https://user-images.githubusercontent.com/25909810/226979450-027b212b-64ee-4ccb-aa54-bc7b729d0a65.png)

Reply:
![image](https://user-images.githubusercontent.com/126335301/226987819-fe461327-7e3a-4ac5-adaf-31a8d9e840fa.png)

Reply:
Looks like for English version of this site locators are different.